src/backend/graphItemService.ts

```diff
@@ -326,7 +326,6 @@ export class GraphItemService {
   private static checkNonEmptyHeaders(headers: unknown): asserts headers is string[] {
     if (
       !Array.isArray(headers) ||
-      headers.length === 0 ||
```

Comment:
what if the first line is empty ?

and how this will impact importing nodes (the need to have an UID)?

Reply:
answered by @renato-rg